UPGRADE.md

```diff
@@ -30,6 +29,52 @@ $entities = $twilio->authy->v1->services("ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                               ->read([], 20);
 ```
 
+[2020-02-19] 6.x.x to 5.x.x
+-----------------------------
+### Overview
+Version `6.x.x` is the first version that officially drops support for Php versions 5.5, 5.6, 7.0, and 7.1.
+
+#### Removal and updates to support removing Php versions 5.5, 5.6, 7.0 and 7.1:
+  - [Drop 'array()' syntax and add return types](https://github.com/twilio/twilio-php/pull/603)
+  - [Drop 'array()' syntax and add return types in generated code](https://github.com/twilio/twilio-php/pull/604)
+  - [Add scalar param type declarations](https://github.com/twilio/twilio-php/pull/605)
+  - [Add scalar param type declarations to generated code](https://github.com/twilio/twilio-php/pull/606)
+  - [Drop support for end of life Php versions](https://github.com/twilio/twilio-php/pull/595)
+
+### CHANGED - Add Return Types
+
+**Added Return Types**
+```php
+// 5.x.x
+public function absoluteUrl($uri) {
+    return \implode('/', array(\trim($this->baseUrl, '/'), \trim($uri, '/')));
+}
+```
+
+```php
+// 6.x.x
+public function absoluteUrl($uri): string {
+    return \implode('/', [\trim($this->baseUrl, '/'), \trim($uri, '/')]);
+}
+```
+
+### CHANGED - Add Scalar Parameters Types
+
+**Added scalar param type**
+```php
+// 5.x.x
+public function absoluteUrl($uri): string {
+    return \implode('/', [\trim($this->baseUrl, '/'), \trim($uri, '/')]);
+}
+```
+
+```php
+// 6.x.x
+public function absoluteUrl(string $uri): string {
+    return \implode('/', [\trim($this->baseUrl, '/'), \trim($uri, '/')]);
+}
+```
```

Comment:
I don't think these need to be called out in the migration guide since they don't require syntactical changes to customer code.

It's more that customers need to be aware that a [`TypeError`](https://www.php.net/manual/en/class.typeerror.php) may _now_ be thrown if they pass an incompatible type into a helper lib function or if they depend on the return type of a helper lib function that is incompatible.